scene/resources/importer_mesh.cpp

```diff
@@ -620,9 +620,7 @@ void ImporterMesh::generate_lods(float p_normal_merge_angle, float p_normal_spli
 						}
 
 						if (!found) {
-							LocalVector<int> new_group;
-							new_group.push_back(corner_idx);
-							normal_group_indices.push_back(new_group);
+							normal_group_indices.push_back({ corner_idx });
```

Comment:
The compiler might by able to eliminate this copy, but the whole thing can be simplified by just constructing the vector in place.